src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
> because it's unique behavior is actually a common pattern that I found in RequestParser

Have you ever thought that there is also error? I now think so.